### PR TITLE
add libavfilter for av package

### PR DESCRIPTION
**Pull request by jeroen**
The `av` package requires libavfilter (part of ffmpeg)

### CODE REVIEW

File: rules/libavfilter.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+          "versions": ["9"]
+        }
+      ]
+    },
+    {
+      "packages": ["ffmpeg-devel"],
+      "pre_install": [
+        { "command": "yum install -y epel-release" },
+        { "command": "yum install -y --nogpgcheck https://mirrors.rpmfusion.org/free/el/rpmfusion-free-release-$(rpm -E %rhel).noarch.rpm https://mirrors.rpmfusion.org/nonfree/el/rpmfusion-nonfree-release-$(rpm -E %rhel).noarch.rpm" }
+      ],
```
Comment (glin) [inline rules/libavfilter.json:46]: Similar note from opencv here - some of the EPEL packages on Rocky Linux 8 seem to require dependencies from PowerTools:
```sh
$ yum install -y ffmpeg-devel
Last metadata expiration check: 0:00:06 ago on Mon Mar 13 18:43:46 2023.
Error: 
 Problem: package ffmpeg-devel-4.2.8-1.el8.x86_64 requires libavdevice(x86-64) = 4.2.8-1.el8, but none of the providers can be installed
  - package ffmpeg-devel-4.2.8-1.el8.x86_64 requires libavdevice.so.58()(64bit), but none of the providers can be installed
  - conflicting requests
  - nothing provides libSDL2-2.0.so.0()(64bit) needed by libavdevice-4.2.8-1.el8.x86_64
(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)
```
Splitting centos8 out into a separate entry with PowerTools should work, e.g. https://github.com/rstudio/r-system-requirements/blob/4921b8c2d1d5c5a2e6b4e4415c11bda21b7a9bcc/rules/gdal.json#L95-L113

**Comment by jeroen** [comment]
Thanks!

**Comment by jeroen** [comment]
Awesome! How long does it take for this to get picked up by e.g.: 

https://packagemanager.rstudio.com/__api__/repos/1/sysreqs?all=false&pkgname=av&distribution=ubuntu&release=20.04

**Comment by glin** [comment]
New rules here are synced daily along with CRAN snapshot updates, so it should be picked up tomorrow. We'll also get binary packages for av and opencv rebuilt after the update, and those binaries should be available immediately if all goes well.

**Pull request merged**